CONTRIBUTING.md

```diff
@@ -72,7 +74,7 @@ Some strings to be translated may contain special characters. Different types of
 
 ### Adding A New Translation
 
-If you're adding a new translation, LOOT's source code must be updated to recognise it. You can do this yourself and include the changes in your translation's pull request if you wish. The files and functions which must be updated are given below.
+If you're adding a new translation, LOOT's source code must be updated to recognise it. While you're welcome to do this yourself and include the changes in your translation's pull request, it's not necessary. You should focus on the translation and following the instructions above; we can handle the rest. The files and functions which must be updated are given below.
```

Comment:
I think "you should" here is a bit strong, I'd say something like "If you're not comfortable making these changes, you can leave them out of your pull request.".

Reply:
That's a fair point, how about this:

```suggestion
If you're adding a new translation, LOOT's source code must be updated to recognise it. While you're welcome to do this yourself and include the changes in your translation's pull request, you don't have to if you're not comfortable with it; we can add it later. The files and functions which must be updated are given below:
```

Also, I just noticed, but the period at the end should probably be a colon since it leads into a list.